front/admin_ui/app.py

```diff
@@ -35,6 +39,23 @@ def healthcheck():
         return f"❌ Failed to connect to {DSS_ENDPOINT} (error {response.status_code})"
 
 
+def draw_graph(width, height):
+    config = ProcessingGraphConfig()
+    libcommon_graph = ProcessingGraph(config.specification)
+    steps = libcommon_graph.steps
+
+    graph = nx.DiGraph()
+    for name, step in steps.items():
```

Comment:
We can also give access to the nx graph as an attribute of libcommon_graph, so that you don't have to recreate the graph (I'm doing this in https://github.com/huggingface/datasets-server/pull/1094, along others things)